code/src/beetle.cpp

```diff
@@ -0,0 +1,75 @@
+#include <beetle.hpp>
+
+namespace r2d2::moving_platform {
+
+    beetle_c::beetle_c(r2d2::usart::usart_ports &usart_port, unsigned int baud_rate,
+             hwlib::pin_out &reset_pin, 
+             base_comm_c &comm):
+             moving_platform_c(comm),
+             qik_2s12v10_motorcontroller (usart_port, baud_rate, reset_pin)
```

Comment:
the `{` should be after the `) ` not on a new line

Reply:
Your commit has fixed this